### ISSUE TITLE
Ubuntu 14.04 cloud image - sudo dpkg -i jobber_1.1-1_amd64.deb fails 

### ISSUE THREAD

**Issue by Tfindelkind** [dshearer/jobber#54]
nutanix@NTNX-AVM:~$ sudo dpkg -i jobber_1.1-1_amd64.deb
Selecting previously unselected package jobber.
(Reading database ... 64029 files and directories currently installed.)
Preparing to unpack jobber_1.1-1_amd64.deb ...
Unpacking jobber (1.1-1) ...
dpkg: dependency problems prevent configuration of jobber:
 jobber depends on init-system-helpers (>= 1.18~); however:
  Version of init-system-helpers on system is 1.14.

dpkg: error processing package jobber (--install):
 dependency problems - leaving unconfigured
Processing triggers for ureadahead (0.100.0-16) ...
Errors were encountered while processing:
 jobber

I used the --force-all to install . It seems to work but is this the right approach?

**Comment by dshearer** [comment]
Yeah...  I kinda suck at making all these packages.  --force-all is the right approach for now.  I'll try to fix this in the future.

**Comment by techtonik** [comment]
Seems fine with 17.04.
```
# dpkg -i jobber_1.2-1_amd64.deb
Selecting previously unselected package jobber.
(Reading database ... 26565 files and directories currently installed.)
Preparing to unpack jobber_1.2-1_amd64.deb ...
Unpacking jobber (1.2-1) ...
Setting up jobber (1.2-1) ...
Adding system user `jobber_client' (UID 112) ...
Adding new user `jobber_client' (UID 112) with group `nogroup' ...
Not creating home directory `/home/jobber_client'.
Setting owners/permissions...
Created symlink /etc/systemd/system/multi-user.target.wants/jobber.service → /lib/systemd/system/jobber.service.
Processing triggers for ureadahead (0.100.0-19) ...
Processing triggers for systemd (232-21ubuntu2) ...
```

**Comment by dshearer** [comment]
Fixed with #156
